src/browser.js

```diff
@@ -39,7 +39,7 @@ var NOT_SUPPORTED = 'The browser target is not supported: %target%';
 module.exports = function (opts) {
 
     opts = opts || {};
-    var target = opts.target || 'chrome';
+    var target = opts.target || 'default';
```

Comment:
Hey @stevengill ... If you make this change, will you bump the version to `2.0.0` since this can potentially break compatibility?

(I say "potentially", but there is a real risk - Visual Studio 2017 uses `cordova-simulate`, which in turn uses `cordova-serve` with the version specified as `^1.0.0`. VS currently does not specify a browser target, relying on the default being Chrome)